databases/usersessions/dataGenerator.js

```diff
@@ -0,0 +1,158 @@
+/* ===========================================================
+Dependencies
+=========================================================== */
+const pg = require('pg');
+const Promise = require('bluebird');
+
+const Chance = require('chance');
+const chance = new Chance();
+
+const topTraders = require('./topTraders.js');
+const config = require('./config.js');
+
+/* ===========================================================
+Connect to PostgreSQL with Pool, using settings in config.js
+=========================================================== */
+
+const pool = new pg.Pool(config);
+
+pool.on('error', (err) => {
+  console.error(err);
+});
+pool.connect();
+
+
+/* ===========================================================
+numberOfSessionBundles
+Defines the number of user visits to simulate
+* Each visit is assigned to one user
+* Each visit contains 1-100 research behaviors
+=========================================================== */
+
+
+const numberOfSessionBundles = 10000;
+
+
+/* ===========================================================
+createSessionBundle
+-generate user_id
+-generate session_id
+-generate number of research requests for this user's visit
+-call sessionResearchActivity
+=========================================================== */
+
+const createSessionBundle = (numberOfSessionBundles) => {
+  let startTime = new Date();
+
+  for (let i = 0; i < numberOfSessionBundles; i++) {
```

Comment:
I'd probably omit the extra newlines here, for readability (per [airbnb](http://airbnb.io/javascript/#whitespace--padded-blocks))